core/timeline_inc.php

```diff
@@ -19,17 +19,36 @@
 
 define( 'MAX_EVENTS', 50 );
 
+# Variables that are defined in parent script:
+#
+# $inc_timeline_filter	Filter array to be used to get timeline event
+#						If undefined, it's initialized as null.
+# $inc_timeline_user		User id to limit timeline scope.
+#						If undefined, it's initialized as null.
+#
+
+if( !isset( $inc_timeline_filter ) ) {
```

Comment:
I have the same doubt.
We don't have a standard for this kind of variables that cross boundaries for included scripts.
I dont feel $g_ is a good prefix, since it can be mixed with general global variables.

Even if they are globals anyway, these are variables that the include script needs to be defined in parent script, an their usage is limited to this include pattern.
As opposed to general globals that are used throughout several api files...

Maybe the comment block at the start of the included script is enough, and keep them as $g_?

Note that we have, in other included scripts, variables with prefix $t_, which is a confusing situation. Having experience that, i tried to make it clear here.